ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -361,6 +389,9 @@ open class SearchBar: UIView {
         // used for cursor or selection handle
         searchTextField.tintColor = style.tintColor
         clearButton.tintColor = style.clearIconColor
+        if let window = window {
+            progressSpinner.color = style.progressSpinnerColor(for: window)
```

Comment:
You need to add this to the ActivityIndicatorView to make sure that the spinner updates its color if it's being displayed while dark and light modes are switched:

    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
        super.traitCollectionDidChange(previousTraitCollection)
        if #available(iOS 13, *) {
            if previousTraitCollection?.userInterfaceStyle != traitCollection.userInterfaceStyle {
                setupLoaderLayer()
            }
        }
    }

Reply:
really? UIColor(light:dark:) didn't do it for you?

Reply:
It doesn't because the ActivityIndicatorView sets a CGColor on its layer instead of a UIColor. So, unfortunately we don't get the color update for free. :(

Reply:
oh yeah great catch!

Reply:
Incorporated the comment and verified that upon switching the mode, the color is getting updated. Thanks

---
In reply to: [470055237](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470055237) [](ancestors = 470055237)

Reply:
Done

---
In reply to: [468750117](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r468750117) [](ancestors = 468750117)